setup.py

```diff
@@ -88,7 +88,7 @@ def run(self):
         "Topic :: Software Development :: Compilers",
         "Topic :: Software Development :: Libraries"
         ],
-    setup_requires=["Opcodes==0.3.9"],
+    setup_requires=["Opcodes==0.3.9", "six"],
```

Comment:
I take that back. You can see from the [test failures](https://travis-ci.org/Maratyszcza/PeachPy/jobs/181234651) that it turns out this is needed in order to run the codegen itself at setup time.